src/index.js

```diff
@@ -8,7 +9,9 @@ makeServer();
 
 ReactDOM.render(
```

Comment:
I see that your package.json has react v18 while you are using react v17 render method. So, better use react v17 to avoid any issues.